features/cellular/framework/AT/ATHandler.cpp

```diff
@@ -256,12 +270,12 @@ void ATHandler::lock()
 
 void ATHandler::unlock()
 {
+    if (_is_fh_usable && (_fileHandle->readable() || (_recv_pos < _recv_len))) {
+        _event_id = _queue.call(Callback<void(void)>(this, &ATHandler::process_oob));
```

Comment:
Syntax nit while you're touching this - I think you can say `callback(this, &ATHandler::process_oob)` to make it a bit simpler-looking.